build/includes/examples.mk

```diff
@@ -27,7 +27,7 @@
 test-examples-on-gcr: example-image-test.autoscaler-webhook example-image-test.cpp-simple
 test-examples-on-gcr: example-image-test.nodejs-simple example-image-test.rust-simple example-image-test.simple-tcp
 test-examples-on-gcr: example-image-test.simple-udp example-image-test.unity-simple example-image-test.xonotic
-test-examples-on-gcr: example-image-test.crd-client
+test-examples-on-gcr: example-image-test.crd-client example-image-test.supertuxkart
```

Comment:
I find stacking multiple things per line confusing. What do you think about doing one per line?

Reply:
Ooh. That looks so much nicer!